kernel/src/main/java/org/kframework/kompile/KompileUsageFormatter.java

```diff
@@ -0,0 +1,401 @@
+// Copyright (c) K Team. All Rights Reserved.
+package org.kframework.kompile;
+
+import com.beust.jcommander.*;
+import com.beust.jcommander.internal.Lists;
+
+import java.util.*;
+import java.util.ResourceBundle;
+
+/**
+ * NOTICE:
```

Comment:
I think you might still need a reference to the original code being Apache 2.0 licensed, and possibly the text of that license. Not 100% sure but worth double checking

Reply:
Updated. How about now?